src/components/Label/Label.tsx

```diff
@@ -0,0 +1,25 @@
+import { LabelProps, Label as PatternflyLabel } from '@patternfly/react-core';
+
+import { EmptyStateSymbol } from 'components/EmptyStateSymbol/EmptyStateSymbol';
+
+interface ILabelMapperObject {
+  text: string;
+  color: LabelProps['color'];
+}
+
+export interface ILabelMapper {
```

Comment:
Create generalized `ILabelMapper`, so that for example `ILabelMapper<Artifact['artifactQuality']>` can be used.

Reply:
done